README.md

```diff
@@ -149,194 +149,9 @@ In the right side, expand the Filters view and then filter by label, specifying
 
 ## Troubleshooting
 
-### Network is unreachable (Slurm V5)
+### Slurm Clusters
```

Comment:
nit: Just for next time, it would have been easier to review if the move was in a separate commit.